activerecord/lib/active_record/migration/compatibility.rb

```diff
@@ -138,6 +138,13 @@ def disable_extension(name, **options)
           super
         end
 
+        def add_foreign_key(from_table, to_table, **options)
```

Comment:
Let's add ` if connection.adapter_name == "PostgreSQL"` condition since this logic is PostgreSQL adapter specific one.